src/patternfly/components/ContextSelector/examples/context-selector.md

```diff
@@ -316,8 +112,14 @@ import './context-selector.css'
 
 ## Documentation
 ### Accessibility
-Added after React implementation.
+| `aria-expanded="false"` | `.pf-c-context-selector__toggle` |  Indicates that the menu is hidden. |
```

Comment:
I think you might need a header row here; it's not displaying as a table.

Reply:
@srambach thanks! Updated.